src/main/java/com/google/gwt/site/webapp/client/GWTProjectEntryPoint.java

```diff
@@ -142,15 +142,20 @@ public boolean f(Element e, int index) {
     GQuery subMenuItem = liParents.last();
 
     subMenuItem.show();
-    String mainNavigationHref = subMenuItem.children("a").attr("href");
+    GQuery section = subMenuItem.children("a");
+    String mainNavigationHref = section.attr("href");
 
     $("#nav a.active").removeClass("active");
     $("#nav a[href='" + mainNavigationHref + "']").addClass("active");
     $("#submenu .active").not(liParents).removeClass("active");
     liParents.add(selectedItem).not(selectedItem.parent()).addClass("active");
 
     // Change the page title for easy bookmarking
-    $("title").text("[GWT] " + subMenuItem.children("a").text() + " - " + selectedItem.text());
+    if (section.isEmpty()) {
+      $("title").text("GWT Project Homepage");
```

Comment:
I would probably remove the word "Homepage".

Currently there are
1. gwtproject.org
2. gwtproject.org/overview.html

which do not have a submenu. So both would show "GWT Project Homepage" even though only 1. should.

In order to fix the `overview.html` page and being able to set a title for it, the code would also need to check `GQuery selectedItem = $("#nav a[href='" + path + "']")` because the current `path` isn't found in a `#submenu` (see line 128).